com.rustdesk.RustDesk.json

```diff
@@ -0,0 +1,118 @@
+{
+  "id": "com.rustdesk.RustDesk",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "rustdesk",
+  "rename-desktop-file": "rustdesk.desktop",
+  "rename-icon": "rustdesk",
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/share/gtk-doc",
+    "/share/man",
+    "*.la",
+    "/share/gtk-doc",
+    "/bin/bsd*",
+    "/bin/xdotool",
+    "/libs/libarchive*"
+  ],
+  "finish-args": [
+    "--share=ipc",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--share=network",
+    "--filesystem=home",
+    "--device=dri",
+    "--socket=pulseaudio",
+    "--talk-name=org.freedesktop.Flatpak"
+  ],
+  "modules": [
+    "shared-modules/libappindicator/libappindicator-gtk3-12.10.json",
+    {
+      "name": "xdotool",
+      "buildsystem": "simple",
+      "build-commands": [
+        "make -j4 && PREFIX=./build make install",
+        "cp -r ./build/*  /app/"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/jordansissel/xdotool/releases/download/v3.20211022.1/xdotool-3.20211022.1.tar.gz",
+          "sha256": "96f0facfde6d78eacad35b91b0f46fecd0b35e474c03e00e30da3fdd345f9ada"
+        }
+      ]
+    },
+    {
+      "name": "pam",
+      "buildsystem": "autotools",
+      "config-opts": [
+        "--disable-static",
+        "--disable-selinux"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/linux-pam/linux-pam/releases/download/v1.3.1/Linux-PAM-1.3.1.tar.xz",
+          "sha256": "eff47a4ecd833fbf18de9686632a70ee8d0794b79aecb217ebd0ce11db4cd0db"
+        }
+      ]
+    },
+    {
+      "name": "libarchive",
+      "buildsystem": "autotools",
+      "config-opts": [
+        "--disable-static"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/libarchive/libarchive/releases/download/v3.5.1/libarchive-3.5.1.tar.gz",
+          "sha256": "9015d109ec00bb9ae1a384b172bf2fc1dff41e2c66e5a9eeddf933af9db37f5a"
+        }
+      ]
+    },
+    {
+      "name": "rustdesk",
+      "buildsystem": "simple",
+      "build-commands": [
+        "bsdtar -zxvf rustdesk.deb",
+        "tar -xvf ./data.tar.xz",
+        "rm -rf usr/share/icons/hicolor/scalable/",
+        "cp -r usr/*  /app/",
```

Comment:
I don't see the metainfo info being installed. You removed it from here, but it is not present in the `.deb`

Reply:
How do you know not installed? I do see installed from log.

Reply:
How can it be installed when it is not in the deb and nowhere in the manifest as a source?

Reply:
![image](https://github.com/flathub/flathub/assets/71636191/fa0e62e4-31ae-431d-b600-0eb538851859)

Reply:
It is closed, you do not need to review any more. https://github.com/flathub/flathub/pull/5233#issuecomment-2106221236

Reply:
Since you seem to know better, neither of the line above show the appstream file. So yeah the appstream file isn't installed.